airflow/config_templates/config.yml

```diff
@@ -1492,6 +1492,14 @@
       type: string
       example: ~
       default: "5"
+    - name: clean_tis_without_dagrun
```

Comment:
Can you write a guides that describes these configuration options and allows us to familiarize ourselves with these options?  The referee documentation is great because it lists all the options, but it's not good documentation for beginners.

Reply:
Created https://github.com/apache/airflow/issues/12193 so we do this before 2.0.0